security/certificates/api-server.adoc

```diff
@@ -11,4 +11,11 @@ cluster CA. Clients outside of the cluster will not be able to verify the
 API server's certificate by default. This certificate can be replaced
 by one that is issued by a CA that clients trust.
 
+[WARNING]
+====
+Make sure to follow the prodcedure xref:../../security/certificates/updating-ca-bundle.adoc#ca-bundle-understanding_updating-ca-bundle[Updating the CA Bundle] before applying custom API named certificate. The cluster must trust CA bundle of the named API certificate, to avoid `x509 signed by unknown authority` errors. Some internal OpenShift components use the `api.<cluster_name>.<base_domain>` endpoint for accessing the Kube API server. If API named certificate is not trusted by cluster, the cluster might become inaccessible.
```

Comment:
```suggestion
Before applying a custom API named certificate, xref:../../security/certificates/updating-ca-bundle.adoc#ca-bundle-understanding_updating-ca-bundle[update the CA bundle certificate]. The cluster must trust the CA bundle of the named API certificate to avoid `x509 signed by unknown authority` errors. Some internal {product-title} components use the `api.<cluster_name>.<base_domain>` endpoint for accessing the Kubernetes API server. If the API named certificate is not trusted by the cluster, the cluster might become inaccessible.
```